recipes/pymatgen/meta.yaml

```diff
@@ -0,0 +1,115 @@
+{% set name = "pymatgen" %}
+{% set version = "2019.7.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 2872f431cb1e9dd64e52f89ec669050ea7e90d2acdfcad4d1fc25b1e1d6e8b19
+
+build:
+  skip: true  # [win or py<35]
+  number: 0
+  script: "{{ PYTHON }} setup.py install --single-version-externally-managed --record record.txt"
+  entry_points:
+    - pmg = pymatgen.cli.pmg:main
+    - feff_input_generation = pymatgen.cli.feff_input_generation:main
+    - feff_plot_cross_section = pymatgen.cli.feff_plot_cross_section:main
+    - feff_plot_dos = pymatgen.cli.feff_plot_dos:main
+    - gaussian_analyzer = pymatgen.cli.gaussian_analyzer:main
+    - get_environment = pymatgen.cli.get_environment:main
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - setuptools
+    - numpy
```

Comment:
is numpy really a build dep?

Reply:
No, I removed it - thanks for the feedback.